introspection/nm-manager.xml

```diff
@@ -381,6 +381,19 @@
     <property name="Version" type="s" access="read"/>
 
     <!--
+        Capabilities:
+
+        The current set of capabilities.  This is a list of 32 bit
+        integers where specific bits are set when a capability is
+        currently available.  For example, capability number 42 would
+        be represented by bit (42 % 32) == 10 in the element with index
+        (42 / 32) == 1 in the array.
+
+        See <link linkend="NMCapability">NMCapability</link> for currently defined capability numbers.
+    -->
+    <property name="Capabilities" type="au" access="read"/>
```

Comment:
> On the other hand, I was wondering if we want to reserve a range of capabilities for downstream users.

This could be done by giving them the odd numbers, and we keep the even ones... But that is probably too clever.